lib/Doctrine/Helpers/UseStatementHelper.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Helpers;
+
+use PHP_CodeSniffer\Files\File;
+use SlevomatCodingStandard\Helpers\UseStatementHelper as UseHelper;
+
+class UseStatementHelper
```

Comment:
I'm not a fan of `Helper` classes or a helper-namespace, but I kept naming and usage of static methods for consistency. If we should use a different approach for outsourcing logic, please let me know.